Source/WebKit/NetworkProcess/Classifier/WebResourceLoadStatisticsStore.h

```diff
@@ -256,8 +260,15 @@ class WebResourceLoadStatisticsStore final : public ThreadSafeRefCounted<WebReso
     HashMap<TopFrameDomain, SubResourceDomain> m_domainsWithCrossPageStorageAccessQuirk;
 
     bool m_hasScheduledProcessStats { false };
-
     bool m_firstNetworkProcessCreated { false };
+
+    struct StorageAccessRequestRecordValue {
+        WebPageProxyIdentifier webPageProxyID;
+        Markable<WallTime> lastRequestTime;
+        WallTime lastLoadTime;
+    };
+    using StorageAccessRequestRecordKey = std::pair<WebCore::FrameIdentifier, RegistrableDomain>;
+    HashMap<StorageAccessRequestRecordKey, StorageAccessRequestRecordValue> m_storageAccessRequestRecords;
```

Comment:
If we move this to NetworkConnectionToWebProcess, then there's less risk of something not cleaning up correctly and introducing an effective leak.